ksqldb/pom.xml

```diff
@@ -45,6 +45,7 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <!--check this -->
```

Comment:
Can you check please if we can update this to the latest version.

Reply:
Updated